core/src/scene/scene.h

```diff
@@ -140,6 +141,8 @@ class Scene {
     std::atomic_ushort pendingTextures{0};
     std::atomic_ushort pendingFonts{0};
 
+    std::vector<SceneError> errors;
```

Comment:
Yeah, I thought it would be good to collect all errors and warnings from scene loading. For now we can change it to hold a single SceneError